src/visualization/client/Turi Create Visualization/src/user_interface/src/elements/Annotate/ErrorBar/index.js

```diff
@@ -13,7 +13,7 @@ class ErrorBar extends Component {
   render() {
     return (
       <div className={style.ErrorBar}>
-        <img src={error} className={style.ErrorIcon}/>
+        <img src={error} className={style.ErrorIcon} alt="Can't Find"/>
```

Comment:
Let's leave the alt text blank (empty string), since the image is purely stylistic and the real content (what a user who is on a screen reader would need to know) is in the following message text.